src/main/groovy/wooga/gradle/unity/tasks/GenerateUpmPackage.groovy

```diff
@@ -132,6 +136,7 @@ class GenerateUpmPackage extends Tar implements BaseSpec, GenerateUpmPackageSpec
 
         from(temporaryDir)
         from(packageDirectory)
+        DirectoryScanner.resetDefaultExcludes()
```

Comment:
This should be inside of a finally block. In case of failure in the previous steps, the excludes will remain changed.